lms/djangoapps/courseware/views.py

```diff
@@ -909,7 +920,17 @@ def course_about(request, course_id):
             'cart_link': reverse('shoppingcart.views.show_cart'),
             'pre_requisite_courses': pre_requisite_courses,
             'course_image_urls': overview.image_urls,
-        })
+        }
+
+        coursetalk_widget_is_enabled = CourseTalkWidgetConfiguration.is_enabled()
+        if coursetalk_widget_is_enabled:
```

Comment:
This bit of logic looks very similar to lines 730 - 738. Can this be factored out in some reasonable way?